src/test/java/hw3/pages/JDITestingDifferentElementsPage.java

```diff
@@ -0,0 +1,56 @@
+package hw3.pages;
+
+import hw3.pages.components.CheckboxRow;
+import hw3.pages.components.LogsSection;
+import hw3.pages.components.RadioButtonsRow;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class JDITestingDifferentElementsPage extends AbstractPage {
+
+    private LogsSection logSection;
+
+    private CheckboxRow checkboxRow;
+
+    private RadioButtonsRow radioRow;
+
+    private WebElement colorsDropdown;
```

Comment:
Why do you decide do not use `@FindBy` annotation?

Reply:
These are instances of classes, not elements. I use @FindBy inside these classes